app/src/main/java/org/wikipedia/descriptions/DescriptionEditRevertHelpView.kt

```diff
@@ -0,0 +1,51 @@
+package org.wikipedia.descriptions
+
+import android.content.Context
+import android.net.Uri
+import android.text.SpannableString
+import android.text.Spanned
+import android.text.TextUtils
+import android.text.method.LinkMovementMethod
+import android.text.style.BulletSpan
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.ScrollView
+import org.wikipedia.R
+import org.wikipedia.WikipediaApp
+import org.wikipedia.databinding.ViewDescriptionEditRevertHelpBinding
+import org.wikipedia.util.DimenUtil
+import org.wikipedia.util.StringUtil
+
+class DescriptionEditRevertHelpView constructor(context: Context, attrs: AttributeSet? = null) : ScrollView(context, attrs) {
+
+    private var qNumber: String = ""
```

Comment:
@cooltey Code convention: We should have the order -> constructo, any variables and then init block if there is.. what do you think..

Reply:
The order should be fine, as we discussed in Slack.